src/main/java/model/Figure.java

```diff
@@ -0,0 +1,12 @@
+package model;
+
+import utility.AreaCalculator;
+import utility.PrintInfo;
+
+public abstract class Figure implements AreaCalculator, PrintInfo {
```

Comment:
The interface `PrintInfo` should not be mixed with `AreaCalculable` if they represent different concepts. It's better to separate concerns and have each interface represent a single responsibility.